app/src/main/java/de/westnordost/streetcomplete/data/StreetCompleteOpenHelper.java

```diff
@@ -302,6 +304,14 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion)
 			db.execSQL(OSM_UNDO_QUESTS_VIEW_CREATE);
 		}
 
+		if(oldVersion < 8 && newVersion >= 8)
+		{
+			db.execSQL("ALTER TABLE " + CreateNoteTable.NAME + " ADD COLUMN " +
+					CreateNoteTable.Columns.IMAGE_PATHS + " text ;");
+			db.execSQL("ALTER TABLE " + OsmNoteQuestTable.NAME + " ADD COLUMN " +
+					OsmNoteQuestTable.Columns.IMAGE_PATHS + " text ;");
+		}
+
```

Comment:
Also, it should be `blob`, not `text`

Reply:
> Did you test this as well?

You mean if everything is stored correctly?

Reply:
No, that I'd expect you to do anyway because this is part of the normal "use this feature from beginning to end once" testing.

I mean testing the SQL database upgrade code.

Reply:
If I upgrade from DB version 7 to 8, a new column called ```image_paths``` is created. I tested this with the "DB Browser for SQLite". Or do you mean something different?

Reply:
Nope, that's fine, thank you! (By the way, for convenience, there are some shell scripts in /tools/ to open it in that DB Browser with one click)

Reply:
I know... This is how I got to know the browser 😉 👍